### DISCUSSION TITLE
Could this work on a Libre LePotato?

### DISCUSSION THREAD

**Discussion by prehprehpreh** [Screenly/Anthias#1823] [Q&A]
I wanna use LePotato for digital signage. I've learned it's not as simple as just loading raspberry pi images onto it.

**Comment by vpetersson** [comment]
It's not a supported board right now. However with the recent changes for the Pi 4, where we moved video to VLC, it's a bit more universal. Having said that, it would likely require some work as there are a number of rpi specific code paths that needs to be reworked.

Contributions are welcome, but it's not something on our internal roadmap.

**Comment by prehprehpreh** [comment]
Thanks for getting back to me. I appreciate the response.

I don't wanna waste your time, but if you have any idea of what I could try
to do to accomplish signage on this, I'd appreciate any info. No problem
though if that's something outside of your scope.

Attached is the operating systems they have available for the device.

Thanks!

**Reply by vpetersson** [reply]
If you have a Debian based OS for the device, then just run the installer and see where it breaks. It might even work. 

Then it's a matter of building the docket image and iterate from there. You likely will need to change the docker base image since the one we use is Raspberry Pi specific. It is however Debian based so should be fairly straight forward to slot in with another image. 

Finally, finding all the code paths that uses pi specific logic. @nicomiguelino would likely have a better idea about that than I do. 

As long as all changes are conditional and doesn't break anything on the Pi we should be able to accept the upstream. 

Good luck!
